daycare/migrations/0006_alter_daycare_area.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.0.3 on 2022-04-27 13:37
```

Comment:
do we need this file?

Reply:
This file auto-created by running makemigration after adding choices to area field.

Reply:
maybe add to 0001 initial the specific field, instead of generating a new file for this
tell me what you think?

Reply:
https://github.com/redhat-beyond/Hotails/pull/65#discussion_r847086629 :bowtie:

Reply:
Yep, thank you for notifying me in this 🙃